app/lib/three_scale/json_representer.rb

```diff
@@ -22,7 +22,7 @@ def to_node(options)
     doc = options.fetch(:doc)
     xml = to_xml(builder: builder)
 
-    Nokogiri::XML::Node.new('root', doc).replace(xml)
+    Nokogiri::XML::Node.new('root', doc).add_child(xml)
```

Comment:
reverted because nokogiry was not supported by ruby 2.4

Reply:
As I understand it, I think `add_child` is OK. Because according to [docs](https://nokogiri.org/rdoc/Nokogiri/XML/Node.html#method-i-add_child) it returns the given node `xml`, but after changing its parent, so this `to_node` method will return an `xml` which is part of `doc` and it has a parent `'root'`:

```
doc
 |_ root
     |_ xml
```

On the other hand, what `replace` did is remove the `'root'` node belonging to `doc` and place `xml` in it's place, so it returned a xml belonging to `doc` with no parent:

```
doc
  |_ xml
```

So they return essentially the same: the difference is now there's a `'root'` node over `xml`, but that doesn't make any test fail.

Reply:
What @akostadinov said

Reply:
> So they return essentially the same: the difference is now there's a 'root' node over xml, but that doesn't make any test fail.

This doesn't sound very convincing. Does API return same or a different thing than before? We must ensure it is the same as before (unless the previous return was obviously broken). Because some things may not be covered by tests.

Reply:
So there is no change in fact. So this is ok with me.